source-container-build/Makefile

```diff
@@ -5,10 +5,6 @@ IMAGE = quay.io/$(ORG)/build-definitions-source-image-build-utils:latest
 image:
 	podman build -t $(IMAGE) .
 
-.PHONY: push
```

Comment:
I suppose you restore it before merge :)

Reply:
It's not actually used in the CI, and I don't think running `make push` locally is a good idea

So I was going to actually remove it

Reply:
This target is not for pushing to the official registry, just for local testing. Without correct credential, pushing to redhat-appstudio does not work actually.